### PR TITLE
v2.1.0

### PR DESCRIPTION
**Pull request by aMarCruz**
Happy new year!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,20 @@
-[![Build Status][travis-image]][travis-url]
-[![NPM version][npm-version-image]][npm-url]
-[![NPM downloads][npm-downloads-image]][npm-url]
-[![MIT License][license-image]][license-url]
-
 # Rollup Plugin for Riot
 
+[![Build Status][travis-badge]][travis-url]
```
Comment (GianlucaGuarini) [inline README.md]: please avoid unnecessary changes like the badge order

File: rollup.config.js [hunk 2]
```diff
@@ -1,11 +1,26 @@
-import buble from 'rollup-plugin-buble'
+const pkg = require('./package.json')
```
Comment (GianlucaGuarini) [inline rollup.config.js]: please use the modules syntax here instead of commonjs

File: .travis.yml [hunk 3]
```diff
@@ -1,6 +1,7 @@
 language: node_js
 node_js:
-  - 8
+  - 'node'
```
[the remark this answers is not in the document]
Comment (GianlucaGuarini) [reply .travis.yml]: I am only asking to remove the `node` item that is not necessary

File: rollup.config.js (hunk 2 above)
Comment (GianlucaGuarini) [reply rollup.config.js]: Ok I didn't know that also because I have never imported the `package.json` in a build but it's fair enough thanks for the explanation

**Comment by aMarCruz** [comment]
Done.

**Comment by GianlucaGuarini** [comment]
thank you
